src/test/groovy/org/mockito/release/internal/gradle/util/ReleaseNotesSerializerIntegrationTest.groovy

```diff
@@ -0,0 +1,39 @@
+package org.mockito.release.internal.gradle.util
+
+import org.mockito.release.notes.internal.DefaultImprovement
+import org.mockito.release.notes.internal.DefaultReleaseNotesData
+import org.mockito.release.notes.vcs.DefaultContributionSet
+import org.mockito.release.notes.vcs.GitCommit
+import spock.lang.Specification
+
+class ReleaseNotesSerializerIntegrationTest extends Specification {
+
```

Comment:
Because in this test `ReleaseNotesSerializer`'s dependencies are not mocked. `DefaultContributionSetSerializerTest` is unit test.